cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -89,9 +89,10 @@ processors:
 
 exporters:
   # Traces
-  sapm:
-    access_token: "${SPLUNK_ACCESS_TOKEN}"
-    endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace"
+  otlp:
+    endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com:443"
```

Comment:
q: why in some places we explicitly have the default HTTPS port and others not? It doesn't matter since 443 the default anyway, just curious if this is intentional or not.

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/issues/9505 for context: right now the code fails if no port is explicitly provided, which really is a bad outcome.

HTTPS means nothing for grpc endpoints, but is used to indicate that a TLS connection is necessary. See https://github.com/open-telemetry/opentelemetry-specification/pull/1729 for more information.